lib/THC/generic/THCTensorMath.cu

```diff
@@ -391,4 +391,42 @@ accreal THCTensor_(trace)(THCState *state, THCTensor *src_) {
   return trace;
 }
 
+#if defined(THC_REAL_IS_FLOAT) || defined(THC_REAL_IS_DOUBLE) || defined(THC_REAL_IS_HALF)
+
+void THCTensor_(linspace)(THCState *state, THCTensor *r_, real a, real b, long n) {
+  THCAssertSameGPU(THCTensor_(checkGPU)(state, 1, r_));
```

Comment:
Thanks! Adding another version of pointwise operator is also my concern when making the change. I would like to go ahead and make the change of these functions. @soumith, may I ask for confirmation with the treatment in non-contiguous Tensors? Thanks!